hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/TestSCMCommonPlacementPolicy.java

```diff
@@ -76,17 +76,30 @@ private void testReplicasToFixMisreplication(
           List<ContainerReplica> replicas,
           DummyPlacementPolicy placementPolicy,
           int expectedNumberOfReplicasToCopy,
-          Map<Node, Integer> expectedNumberOfCopyOperationFromRack) {
+          Map<Node, Integer> expectedNumberOfCopyOperationFromRack)
```

Comment:
I might not see it, bit could we add a test to vaildate the new functionality, which is not selecting replicas with "can copy" set to false. Ideally we should test out the scenario where we cannot find enough to copy too, and validate it does not fail but has partial success.

Reply:
yeah, I have added that currently adding test cases for the ECReplicationHandler

Reply:
Done